e2e/cypress/integration/system_console/settings_spec.js

```diff
@@ -0,0 +1,241 @@
+                // # Create SAML user
+                samlUser = user2;
+                const body = {
+                    from: 'email',
+                    auto: false,
+                };
+                body.matches = {};
+                body.matches[user2.email] = user2.username;
+
+                return migrateAuthToSAML(body);
+            }).then(() => {
+                return cy.apiGenerateMfaSecret(mfaUser.id);
+            }).then((res) => {
+                // # Create MFA user
+                const token = authenticator.generateToken(res.code.secret);
+
+                return cy.apiActivateUserMFA(mfaUser.id, true, token);
+            });
+    });
+
+    it('MM-T953: Verify correct authentication method', () => {
+        cy.visit('/admin_console/user_management/users');
+
+        // # Type sysadmin
+        cy.get('#searchUsers').clear().type('sysadmin').wait(TIMEOUTS.HALF_SEC);
+
+        // * Verify sign-in method
+        cy.findByTestId('userListRow').within(() => {
+            cy.get('.more-modal__details').
+                should('be.visible').
+                and('contain.text', 'Sign-in Method: Email');
+        });
+
+        // # Type saml user
+        cy.get('#searchUsers').clear().type(samlUser.username).wait(TIMEOUTS.HALF_SEC);
+
+        // * Verify sign-in method
+        cy.findByTestId('userListRow').within(() => {
+            cy.get('.more-modal__details').
+                should('be.visible').
+                and('contain.text', 'Sign-in Method: SAML');
+        });
+
+        // # Type ldap user
+        cy.get('#searchUsers').clear().type(ldapUser.username).wait(TIMEOUTS.HALF_SEC);
+
+        // * Verify sign-in method
+        cy.findByTestId('userListRow').within(() => {
+            cy.get('.more-modal__details').
+                should('be.visible').
+                and('contain.text', 'Sign-in Method: LDAP');
+        });
+
+        // # Type mfa user
+        cy.get('#searchUsers').clear().type(mfaUser.username).wait(TIMEOUTS.HALF_SEC);
+
+        // * Verify sign-in method
+        cy.findByTestId('userListRow').within(() => {
+            cy.get('.more-modal__details').
+                should('be.visible').
+                and('contain.text', 'MFA: Yes');
+        });
+    });
+
+    it('MM-T1149: Hide mobile-specific settings', () => {
+        // * Ensure license
+        cy.apiRequireLicense();
+
+        // # Visit license page
+        cy.visit('/admin_console/about/license');
+
+        // # Remove license
+        cy.get('#remove-button').click();
+
+        cy.visit('/admin_console/site_config/file_sharing_downloads');
+
+        // * Check buttons
+        cy.get('#adminConsoleWrapper .wrapper--fixed > .admin-console__wrapper').
+            should('be.visible').
+            and('contain.text', 'Allow File Sharing');
+
+        // Bring back license
+        cy.apiRequireLicense();
+    });
+
+    it('MM-T1161: Data retention - Settings are saved', () => {
```

Comment:
This and the last 2 test cases can remain together but please move to `/enterprise` since it'e enterprise only. Also, in the `before` hook, please add `cy.apiRequireLicense()`.